scripts/dll/webpack.config.dll.js

```diff
@@ -143,7 +143,14 @@ const webpackConfig = {
 			},
 			{
 				test: /\.ts$/,
-				use: [ 'ts-loader' ]
+				use: [
+					{
+						loader: 'ts-loader',
+						options: {
+							configFile: path.resolve( process.cwd(), 'tsconfig.test.json' )
```

Comment:
Are you sure that this is the correct file?

Reply:
This is used to build production dlls so I would suggest to use a separate tsconfig file for this. Maybe `tsconfig.dll.json`? @pomek WDYT?

Reply:
https://github.com/ckeditor/ckeditor5/pull/13615#discussion_r1126915268

Reply:
@pomek comment that you mentioned does not apply to this case. This script is building base DLL and this is not checking whether file exists or not (as in the other comment). What I want to say is that we should not use `dev` tsconfig for production DLL build just to avoid any future WTFs when change in `dev` would affect production DLLs.

Reply:
Production DLL is built from development sources, so TBH, it seems to be safe.

Webpack and TypeScript configurations do not land on npm. If somebody wants to refresh the build, it must be done using the `ckeditor5` repository representing our development environment. Perhaps I'm missing something, and I would ask what can go wrong here.

Reply:
Btw, what would the `tsconfig.dll` file contain?